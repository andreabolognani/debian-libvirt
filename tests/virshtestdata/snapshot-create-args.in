snapshot-create-as --print-xml test name desc vda vdb
snapshot-create-as --print-xml test name desc --diskspec vda vdb
snapshot-create-as --print-xml test name desc --diskspec vda --diskspec vdb
snapshot-create-as --print-xml test name desc vda vdb
snapshot-create-as --print-xml test --diskspec vda name --diskspec vdb desc
snapshot-create-as --print-xml --description desc --name name --domain test vda vdb
snapshot-create-as --print-xml --description desc --diskspec vda --name name --domain test vdb
snapshot-create-as --print-xml test name vda vdb
snapshot-create-as --print-xml test --diskspec vda,file=a&b,,c,snapshot=external --description 1<2 --diskspec vdb --memspec file=d,,e
