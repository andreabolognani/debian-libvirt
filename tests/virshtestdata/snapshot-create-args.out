<domainsnapshot>
  <name>name</name>
  <description>desc</description>
  <disks>
    <disk name='vda'/>
    <disk name='vdb'/>
  </disks>
</domainsnapshot>

<domainsnapshot>
  <name>name</name>
  <description>desc</description>
  <disks>
    <disk name='vda'/>
    <disk name='vdb'/>
  </disks>
</domainsnapshot>

<domainsnapshot>
  <name>name</name>
  <description>desc</description>
  <disks>
    <disk name='vda'/>
    <disk name='vdb'/>
  </disks>
</domainsnapshot>

<domainsnapshot>
  <name>name</name>
  <description>desc</description>
  <disks>
    <disk name='vda'/>
    <disk name='vdb'/>
  </disks>
</domainsnapshot>

<domainsnapshot>
  <name>name</name>
  <description>desc</description>
  <disks>
    <disk name='vda'/>
    <disk name='vdb'/>
  </disks>
</domainsnapshot>

<domainsnapshot>
  <name>name</name>
  <description>desc</description>
  <disks>
    <disk name='vda'/>
    <disk name='vdb'/>
  </disks>
</domainsnapshot>

<domainsnapshot>
  <name>name</name>
  <description>desc</description>
  <disks>
    <disk name='vda'/>
    <disk name='vdb'/>
  </disks>
</domainsnapshot>

<domainsnapshot>
  <name>name</name>
  <description>vda</description>
  <disks>
    <disk name='vdb'/>
  </disks>
</domainsnapshot>

<domainsnapshot>
  <description>1&lt;2</description>
  <memory file='d,e'/>
  <disks>
    <disk name='vda' snapshot='external'>
      <source file='a&amp;b,c'/>
    </disk>
    <disk name='vdb'/>
  </disks>
</domainsnapshot>

